architecture/core_concepts/projects_and_users.adoc

```diff
@@ -130,6 +131,16 @@ endif::openshift-enterprise,openshift-origin[]
 ifdef::openshift-dedicated,openshift-aro,openshift-online[]
 their own projects.
 endif::openshift-dedicated,openshift-aro,openshift-online[]
+endif::openshift-dedicated,openshift-enterprise,openshift-online,openshift-origin[]
+
+ifdef::openshift-aro[]
+{product-title} is a managed service. The cluster administrator role is not available to customers; it is delegated to the customer for cluster-managed operations.
```

Comment:
```suggestion
{product-title} is a managed service. The cluster administrator role is not available to customers. The customer administrator role is delegated to the customer for cluster-management operations.
```
Sorry, I wrote it in a convoluted way in the original.
There are two distinct roles: "cluster-admin" which can do anything and is not available to customers,
"customer-admin" (implemented as roles called "customer-admin-cluster" and "customer-admin-project") role which is available to customers but has limited permissions.